backend/src/main/kotlin/dev/dres/api/rest/handler/evaluation/scores/TeamGroupScoreHandler.kt

```diff
@@ -44,7 +44,7 @@ class TeamGroupScoreHandler : AbstractScoreHandler(), GetRestHandler<List<ApiTea
         }
 
 
-            val rac = ctx.runActionContext()
+//            val rac = ctx.runActionContext()
             /* TODO: Not suite sure where the teamGroupAggregator got lost.*/
```

Comment:
what about this?

Reply:
I'll investigate...

Reply:
Fixed in [c8eedfb](https://github.com/dres-dev/DRES/pull/460/commits/c8eedfba9b3ea084da93e29ebdde0022986577de)